lua_examples/telnet/telnet.lua

```diff
@@ -143,11 +144,11 @@ return {
       wifi.setmode(wifi.STATION, false)
       wifi.sta.config { ssid = ssid, pwd  = pwd, save = false }
     end
-    tmr.alarm(0, 500, tmr.ALARM_AUTO, function()
+    local tmr_wait = tmr.create()
```

Comment:
A nit pick but I find `tmr_wait` a subpar name.

Reply:
renamed to `t` so it's compatible with #2650